.images/architecture.svg

```diff
@@ -1,3 +1,3 @@
: unsafe center; width: 1px; height: 1px; padding-top: 977px; margin-left: 1232px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 11.8px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #ffffff; white-space: nowrap; ">Current ClamAV vulnerability <br />definitionsare fetched from an<br />S3 bucket at run time.</div></div></div></foreignObject><text x="1232" y="981" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">C...</text></switch></g><path d="M 1185 1091 L 963 1091" fill="none" stroke="#333333" stroke-miterlimit="10" pointer-events="stroke"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 1094px; margin-left: 1075px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 11.8px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #ffffff; white-space: nowrap; ">On each file upload or update,<br />and S3 Event triggers a Lambda<br />that runs ClamAV against the file<br />to check for viruses/malware.</div></div></div></foreignObject><text x="1075" y="1098" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">O...</text></switch></g><path d="M 452 1416 L 643 1416 Q 655 1416 655 1428 L 655 1976 Q 655 1988 667 1988 L 875 1988" fill="none" stroke="#333333" stroke-miterlimit="10" pointer-events="stroke"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 1991px; margin-left: 770px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 11.8px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #ffffff; white-space: nowrap; ">/prince POST</div></div></div></foreignObject><text x="770" y="1995" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">/...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
@nnti on this image, please update the new proxy api to be labeled `proxy` and not `prince`